tests/utils/runtime/errors.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package runtime
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+)
+
+// ErrorsFanIn takes care of fanning runtime errors from
```

Comment:
Can you explain why this needs to be exported? I would have assumed the `Nodes` type just handles this internally.

Reply:
I moved it to `utils/nodes` as unexported.

It was the Nodes initially handling this internally but that got messy rather quickly, so I split it away in that struct.
I then moved it to its own package to avoid polluting the namespace in utils/nodes, but that might had been overkill indeed.